### DISCUSSION TITLE
Group Variable

### DISCUSSION THREAD

**Discussion by Umhead22** [ScratchAddons/ScratchAddons#7726] [Ideas]
It will be much easier if Scratcher could make a group of variables/list, it will be much useful on large projects that uses a lot of variables
![Screenshot 2024-08-14 16 03 41](https://github.com/user-attachments/assets/e28b541b-ba2e-44df-af17-ebe116a750d3)
example

**Comment by Samq64** [comment]
Does [data category tweaks](https://scratch.mit.edu/scratch-addons-extension/settings#addon-data-category-tweaks-v2) solve your problem?

**Reply by Umhead22** [reply]
No

**Comment by SiIentNinja** [comment]
While we're at it, I think it could also be useful to filter by category, especially for projects with a large number of variables (the example image) you will need to scroll down really far.
